client/my-sites/themes/themes-selection.jsx

```diff
@@ -160,6 +166,7 @@ const ThemesSelection = React.createClass( {
 					isPurchased={ this.props.isThemePurchased }
 					isInstalling={ this.props.isInstallingTheme }
 					loading={ this.props.isRequesting }
+					emptyContent={ this.showNoThemesFound() }
```

Comment:
We _could_ take advantage of the fact that this `emptyContent` prop already exists, and pass the empty `<div/>` all the way down in `emptyContent` props from `single-site-jetpack.jsx` (instead of the `showNoThemesFound` boolean props).

There are a couple of reasons why this might be better:
* `showNoThemesFound` is hard to name
* We can keep the `null : <div />` logic in `single-site-jetpack.jsx` along with the related logic

I'm not thoroughly convincing myself with this suggestion, however, so feel free to completely ignore it.